Code/GraphMol/Abbreviations/Wrap/testAbbreviations.py

```diff
@@ -123,9 +123,17 @@ def testAbbreviationsSubstanceGroups(self):
 M  V30 END CTAB
 M  END''')
     nm = rdAbbreviations.CondenseAbbreviationSubstanceGroups(m)
-    nm.RemoveAllConformers() # avoid coords in CXSMILES
+    nm.RemoveAllConformers()  # avoid coords in CXSMILES
```

Comment:
Is this commit supposed to be in this batch?

Reply:
that's line is just a formatting change that `yapf` did.
The next delta in this file (at line 129) is the test for the bug fix